service/src/tests/event.rs

```diff
@@ -39,140 +39,6 @@ macro_rules! test_with_dbs {
     }
 }
 
-test_with_dbs!(
```

Comment:
moved to store crate because we use predefined event ID ranges to assert the hash. at this layer we can't build a random body for a CID but we can insert raw data at the store crate and run the queries there